src/components/Navigation/Search.scss

```diff
@@ -50,11 +54,36 @@
 
   .algolia-docsearch-suggestion {
     padding: 0;
+    .algolia-docsearch-suggestion--subcategory-column {
```

Comment:
Wonder if the notify when they change these classes.

Reply:
I'm afraid they won't since this seems impossible to be done. But if they stick to semver, we might be safe.